Cargo.toml

```diff
@@ -390,7 +390,24 @@ maintainer           = "Konstantin Sivakov <konstantin@datHere.com>"
 copyright            = "2024, datHere Inc. <www.dathere.com>"
 extended-description = """A high performance CSV data-wrangling toolkit."""
 depends              = "$auto"
-features             = ["feature_capable"]
 section              = "utility"
 priority             = "optional"
-assets               = [["target/release/qsv", "/usr/local/bin/", "755"]]
+
+# Default feature and asset
+features             = ["feature_capable"]
```

Comment:
You may want to add the features you actually want to enable. Otherwise, this is just like qsvlite.
I suggest to add everything EXCEPT python, as doing so creates a runtime dependency on the specific version of python libraries being present at startup, even if you're not using the `py` command:

`["feature_capable","apply","fetch","foreach","geocode","lens","luau","polars","prompt","to"]`

You can also enable the `python` feature if you can specify cargo-deb to have a dependency on the specific version of python you need to be installed in the config.